### PR TITLE
Improve the naming of the element argument hash index selector

### PR DESCRIPTION
**Pull request by optimumtact**

### CODE REVIEW

File: code/__DEFINES/dcs/flags.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 /// Causes the detach proc to be called when the host object is being deleted
 #define ELEMENT_DETACH (1 << 0)
 /**
- * Only elements created with the same arguments given after `id_arg_index` share an element instance
+ * Only elements created with the same arguments given after `argument_hash_start_idx` share an element instance
```
Comment (optimumtact) [inline code/__DEFINES/dcs/flags.dm:15]: is this a lie? it's either after or inclusive @ninjanomnom

**Review by Mothblocks — APPROVED** [review]
I hate this var

**Comment by JohnFulpWillard** [comment]
draft for conflicts

**Comment by optimumtact** [comment]
CONFLICTS HOW

**Comment by Mothblocks** [comment]
remake this later

**Comment by optimumtact** [comment]
YOU CONFLICTED IT!

**Pull request closed without merging**